docs/learn/prompts.md

```diff
@@ -10,76 +10,49 @@ Prompts are the foundation of effective communication with Large Language Models
 
 The primary means of writing prompts in Mirascope is through prompt templates, which are just formatted strings (as they should be) with a few additional conveniences. This allows you to define prompts such that they are dynamic and reusable.
```

Comment:
note: we should add a second section `Messages` where we discuss writing the messages style of prompting as described in #490 

In this section, we should link to the `Messages` section with a blurb saying something like "If you are not a fan of prompt templates and want to write prompts using formatted strings directly, see the Messages section below" and add a quick link

Reply:
i would then also mention early on in this new section that you can write prompts as prompt templates with additional templating conveniences if you'd like, see the section below or something along those lines